wp-content/themes/csisnuclearnetwork/assets/_scss/components/_footnotes.scss

```diff
@@ -7,7 +7,10 @@
 .single__content {
   .easy-footnote {
     // font-family: $font__merriweather;
-    font-size: rem(15px);
+    font-size: rem(12px);
+    text-transform: uppercase;
+    vertical-align: super;
```

Comment:
We don't need `text-transform` or `vertical-align` here, so they can be removed.